easybuild/easyconfigs/g/GraphicsMagick/GraphicsMagick-1.3.45-GCCcore-13.3.0.eb

```diff
@@ -40,12 +40,29 @@ dependencies = [
     ('Ghostscript', '10.03.1'),
 ]
 
+# When building Octave, mixing rpathed shared libraries with libGraphicsMagick++.a will cause the check on the library
+# to fail due to missing symbols. The default previously was to build the static library only.
+configure_opts = '--enable-shared --enable-static'
+
 modextrapaths = {MODULE_LOAD_ENV_HEADERS: ['include/GraphicsMagick']}
 
 sanity_check_paths = {
-    'files': ['bin/gm', 'lib/libGraphicsMagick.a', 'lib/libGraphicsMagick++.a',
-              'lib/libGraphicsMagickWand.a'],
+    'files': [
+        'bin/gm',
+        'lib/libGraphicsMagick.a',
+        'lib/libGraphicsMagick++.a',
+        'lib/libGraphicsMagickWand.a',
+        'lib/libGraphicsMagick.so',
+        'lib/libGraphicsMagick++.so',
+        'lib/libGraphicsMagickWand.so'
+    ],
     'dirs': ['include/GraphicsMagick', 'lib/pkgconfig'],
 }
 
+sanity_check_commands = [
+    "pkg-config --cflags GraphicsMagick",
+    "pkg-config --cflags GraphicsMagick++",
+    "pkg-config --cflags GraphicsMagickWand",
+]
```

Comment:
you're relying on the OS having pkg-config installed here

Reply:
For now I've added the `pkgconf` to the normal dependencies (https://github.com/easybuilders/easybuild-easyconfigs/pull/22930/commits/70e1fd8e766e1d7c844041a34a4dd83b174dacd3), by looking at the code I do not see an easy way to have an equivalent of `builddependencies` for sanity checks.
As far as i understand, the dependencies for the sanity check are pulled in through the creation of a fake module, which filters out the deps marked as `build_only`

Reply:
Yeah, build dependencies won't be loaded anymore during sanity check (and that's intentional).

I guess we could consider introducing a concept like `sanity_check_deps`, i.e. extra module that should be loaded only during the sanity check (not by the module that's generated).

You could look into https://github.com/easybuilders/easybuild-framework/pull/4766 for inspiration for that

Reply:
I think it would just suffice to check that the .pc files exist rather than complicating things.

Reply:
maybe just do a `grep` on `libtiff-4` in the pc file?